ui/app/AppLayouts/Chat/CommunityComponents/CommunityProfilePopup.qml

```diff
@@ -156,6 +169,16 @@ ModalPopup {
                 stack.pop()
             }
         }
+
+        StatusButton {
+            text: qsTr("Invite")
+            anchors.right: parent.right
+            enabled: stack.currentItem.contactListSearch !== undefined && stack.currentItem.contactListSearch.pubKeys.length > 0
+            onClicked: {
+                stack.currentItem.sendInvites(stack.currentItem.contactListSearch.pubKeys)
+                stack.pop()
+            }
+        }
```

Comment:
This is rather unfortunate but made due to limitations in how we've implemented our modal...

There's a specific button that needs to be rendered in the modal only when a certain section of the stackview inside the modal is active and only if a certain condition is true. We have to deep lookup into `stack.currentItem` here to make this work.

Something we should revisit in the future to see if this can be solved nicely in a decouple fashion.

In the worst case, we can always extract entire modals (meaning header +content + footer) and have those rendered inside the actual modal so that we have full control over CTAs within footers and header customization, per view.